.github/workflows/validate.yml

```diff
@@ -12,6 +12,11 @@ on:
       - 'alpha'
       - '!all-contributors/**'
   pull_request: {}
+
+permissions:
+  actions: write #  to cancel/stop running workflows (styfle/cancel-workflow-action)
```

Comment:
> It is recommended to have most strict permissions on the top level

Should we disable all permissions by default and enable the permissions on a job basis?

Reply:
Yes, sometime it makes sense to move the common denominator to the top though. I have updated the pull request.